<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>613</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabSource1">
       <attribute name="icon">
        <iconset>
         <normaloff>:/Icons/Store.png</normaloff>:/Icons/Store.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Источник №1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="FormSourceStore" name="formSourceStore1" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDest">
       <attribute name="icon">
        <iconset resource="StorageUi.qrc">
         <normaloff>:/Icons/Communism.png</normaloff>:/Icons/Communism.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Конечная цель</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelError">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);
font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Проблемы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FormDestStore" name="formDestStore" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionAddSource"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddSource">
   <property name="icon">
    <iconset resource="../../ArmK/AdminUi/Resource.qrc">
     <normaloff>:/ObjTree/rep</normaloff>:/ObjTree/rep</iconset>
   </property>
   <property name="text">
    <string>Добавить источник</string>
   </property>
   <property name="toolTip">
    <string>Добавить источник</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormSourceStore</class>
   <extends>QWidget</extends>
   <header>FormSourceStore.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormDestStore</class>
   <extends>QWidget</extends>
   <header>FormDestStore.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ArmK/AdminUi/Resource.qrc"/>
  <include location="../../ArmK/AdminUi/Resource.qrc"/>
  <include location="../../ArmK/AdminUi/Resource.qrc"/>
  <include location="../../ArmK/AdminUi/Resource.qrc"/>
  <include location="StorageUi.qrc"/>
  <include location="../../ArmK/AdminUi/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
